pkg/lint/rules/deprecations.go

```diff
@@ -0,0 +1,63 @@
+/*
+Copyright The Helm Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package rules // import "helm.sh/helm/v3/pkg/lint/rules"
+
+import "fmt"
+
+// deprecatedAPIs lists APIs that are deprecated (left) with suggested alternatives (right).
+//
+// An empty rvalue indicates that the API is completely deprecated.
+var deprecatedAPIs = map[string]string{
+	"extensions/v1 Deployment":             "apps/v1 Deployment",
+	"extensions/v1 DaemonSet":              "apps/v1 DaemonSet",
+	"extensions/v1 ReplicaSet":             "apps/v1 ReplicaSet",
+	"extensions/v1beta1 PodSecurityPolicy": "policy/v1beta1 PodSecurityPolicy",
+	"extensions/v1beta1 NetworkPoliicy":    "networking.k8s.iio/v1beta1 NetworkPolicy",
```

Comment:
iio --> io

Reply:
`Ingress` in `extensions/v1beta1` will go to `networking.k8s.io/v1beta1`

Reply:
I was so impressed that @hickeyma's eye was able to catch the double 'i', that I got curious about this change.

I think @technosophos 'i' key is sticky 'cause: NetworkPoliicy --> NetworkPolicy 😀

Reply:
Yes -- I have a MacBook with a broken keyboard. My `i`'s and `t`'s tend to double and `r`'s tend to double or triple. I can usually spot the r and t mistakes. But the i's are harder for me to pick out.

Reply:
In fact, in Slack you can usually tell whether I am using the built-in or external keyboard by these tells.